test/datadog-operator/baseline/Operator_Deployment_default.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: datadog-agent
   labels:
     app.kubernetes.io/name: datadog-operator
-    helm.sh/chart: datadog-operator-2.9.0
+    helm.sh/chart: datadog-operator-2.9.1
```

Comment:
This is the result of running `make update-test-baselines`. Since [2.9.1](https://github.com/DataDog/helm-charts/blob/main/charts/datadog-operator/Chart.yaml#L3) is the latest version of the Operator chart, I decided to commit this change.